helm-charts/helm3/strimzi-kafka-operator/values.yaml

```diff
@@ -10,7 +10,7 @@ watchAnyNamespace: false
 
 defaultImageRegistry: quay.io
 defaultImageRepository: strimzi
-defaultImageTag: 0.37.0
+defaultImageTag: 0.38.0
```

Comment:
The point is, that this file should not be changed at all. You can change only the files in `./packaging/`.

Reply:
thanks a lot @scholzj. I was a little bit confused and I am clear now. Just updated the PR and please review. 👍